charts/tidb-backup/templates/scripts/_start_restore.sh.tpl

```diff
@@ -1,4 +1,4 @@
-set -euo pipefail
+set -uo pipefail
```

Comment:
why remove `-e`?

Reply:
The return code of ```mysql -u root -h 172.16.4.32 -P 32438 -pLouis456 -e 'select version();'``` command is 1 if tidb is not available.

Reply:
@tennix PTAL

Reply:
It's useless to execute loader if the download backup failed, but for the test connection part, we can skip error checks. The `set -e` can be set to specific part IIRC.

Reply:
https://superuser.com/questions/940498/how-to-disable-set-e-for-an-individual-command